hauberk.app.domain-verification.json

```diff
@@ -0,0 +1,53 @@
+{
+    "providerId": "hauberk.app",
+    "providerName": "Hauberk",
+    "serviceId": "domain-verification",
+    "serviceName": "Hauberk Domain Verification",
+    "version": 1,
+    "logoUrl": "https://assets.hauberk.app/hauberkLogo.svg",
+    "syncPubKeyDomain": "hauberk.app",
+    "description": "Configures DNS records for Hauberk domain verification",
+    "variableDescription": "unique is randomly generated for use in the username part of the dmarc rua email address; publicKey is a cryptographic signature for signing DKIM; region is an AWS region.",
+    "syncBlock": false,
+    "syncRedirectDomain": "hauberk.app",
+    "records": [
+        {
+            "groupId": "outbound",
+            "type": "MX",
+            "host": "send",
+            "pointsTo": "feedback-smtp.%region%.amazonses.com",
+            "priority": 10,
+            "ttl": 300
+        },
+        {
+            "groupId": "outbound",
+            "type": "SPFM",
+            "host": "send",
+            "spfRules": "include:amazonses.com"
+        },
+        {
+            "groupId": "outbound",
+            "type": "TXT",
+            "host": "hauberk._domainkey",
+            "data": "%publicKey%",
+            "ttl": 300
+        },
+       {
+            "groupId": "outbound",
+            "type": "TXT",
+            "host": "_dmarc.%domain%.",
```

Comment:
For DMARC I would recommend using essential: false.
Just if someone would try to edit DMARC later manually the whole template would be dropped with some providers (IONOS, GoDaddy) if the record is essential.

Reply:
Ok, that sounds good. I did the same for the inbound record as well.

If this all looks good now, I was thinking I should add 3 additional templates to allow someone to choose the level of services:

- This template: configures outbound, inbound email with dmarc
- Outbound only template
- Outbound + inbound template
- Outbound + dmarc only template

Depending on what options they select when configuring the domain it would choose the template to use, ie.:

Enter your domain name
|_________________________|

- [ ] Process inbound email?
- [ ] Monitor dmarc?


Does that make sense to do it that way?